packages/aws-cdk/lib/api/hotswap/lambda-functions.ts

```diff
@@ -232,25 +233,17 @@ class LambdaFunctionHotswapOperation implements HotswapOperation {
     const operations: Promise<any>[] = [];
 
     if (resource.code !== undefined) {
-      const updateFunctionCodePromise = lambda.updateFunctionCode({
+      const updateFunctionCodeResponse = await lambda.updateFunctionCode({
         FunctionName: this.lambdaFunctionResource.physicalName,
         S3Bucket: resource.code.s3Bucket,
         S3Key: resource.code.s3Key,
         ImageUri: resource.code.imageUri,
         ZipFile: resource.code.functionCodeZip,
       }).promise();
 
-      // only if the code changed is there any point in publishing a new Version
```

Comment:
Let's keep this comment (it should be now on line 246 after the changes in this PR).